src/sites/doc/components/search/search.tsx

```
@@ -0,0 +1,96 @@
+      index++;
+    }
+    if (e.keyCode == 38) {
+      index--;
+    }
+    if (searchIndex < 0) {
+      index = 0;
+    }
+    if (searchList.length > 0) {
+      const cName = searchList[index] && searchList[index].name;
+      if (cName) {
+        setSearchCurName(cName);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)